pkg/log/log.go

```diff
@@ -160,6 +160,17 @@ func getStringStat(getFunc func() (string, error)) string {
 	return stat
 }
 
+// getStringSliceStat is a small wrapper around gopsutil/process functions
+// to return the stat if available, or an error message if not, so
+// that either way the info will be captured in the log.
+func getStringSliceStat(getFunc func() ([]string, error)) string {
+	stat, err := getFunc()
+	if err != nil {
+		return fmt.Sprintf("could not get stat: %v", err)
+	}
+	return stat[0]
```

Comment:
```suggestion
	// We only use this function for `Status` at the moment, which is guaranteed to have one element when successful.
	return stat[0]
```

Just adding what you have in the MR description so that it doesn't get lost/we don't misuse this in the future!